### PR TITLE
Added Main.CSS

### PR DESCRIPTION
**Pull request by Federicofilini**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
           </div>
         </article>
       </section>
+      <!-- ----------------------------------------------------Blog--- -->
+      <section class="Blog_section">
+        <div class="Blog_header">
+          <h2>Our blog</h2>
+        </div>
```
Comment (andacenko) [inline index.html:201]: `<h2 class="blog__header">text</h2>`

File: index.html [hunk 2]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
           </div>
         </article>
       </section>
+      <!-- ----------------------------------------------------Blog--- -->
+      <section class="Blog_section">
+        <div class="Blog_header">
```
Comment (andacenko) [inline index.html:199]: delete this div

File: index.html [hunk 3]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
           </div>
         </article>
       </section>
+      <!-- ----------------------------------------------------Blog--- -->
+      <section class="Blog_section">
```
Comment (andacenko) [inline index.html:198]: 'blog__section' or just 'blog'

File: index.html [hunk 4]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        <div class="Blog_header">
+          <h2>Our blog</h2>
+        </div>
+        <div class="Blog_Grid_section">
+          <div class="Blog_kard">
+            <div class="blog_foto_box">
+              <div class="Nft_box">
```
Comment (andacenko) [inline index.html:205]: delete "Nft_box"

File: index.html [hunk 5]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+                <p>NFT</p>
```
Comment (andacenko) [inline index.html:206]: just `<span class="blog__label">NFT</span>`

File: index.html [hunk 6]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              <img src="" alt="" />
+            </div>
+            <div class="Blog_card_info">
+              <div class="Box-info1">
+                <img src="" alt="" />
+                <p>Tony Nguyen</p>
+              </div>
+              <div class="Box-info1">
+                <img src="" alt="" />
+                <time datetime=""></time>
```
Comment (andacenko) [inline index.html:217]: empty block

File: index.html [hunk 7]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              </div>
+            </div>
+            <div class="Blog_header_box">
```
Comment (andacenko) [inline index.html:220]: delete "Blog_header_box"

File: index.html [hunk 8]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              <h2 class="Blog_header">Sonic live</h2>
+              <p>
+                One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
+                nulla imperdiet...
+              </p>
+            </div>
+            <div class="Blog_Button_box">
```
Comment (andacenko) [inline index.html:227]: delete "Blog_Button_box"

File: index.html [hunk 9]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              <a href="" class="Button_readMore"
```
Comment (andacenko) [inline index.html:228]: 'btn'

File: index.html [hunk 10]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+                >Read more<img src="" alt=""
+              /></a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- ---------------------------------------------------Abaut Us -->
+      <section class="AboutUs">
```
Comment (andacenko) [inline index.html:386]: 'about'

File: index.html [hunk 11]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        <div class="AbautUs--Header_box">
```
Comment (andacenko) [inline index.html:387]: delete "AbautUs--Header_box"

File: index.html [hunk 12]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <h2 class="AboutUs_heager">About us</h2>
```
Comment (andacenko) [inline index.html:388]: 'about__header'

File: index.html [hunk 13]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <p>
+            Duis aute irure dolor in reprehenderit in voluptate velit esse
+            cillum dolore eu fugiat nulla pariatur. Excepteur sint occae cat
+            cupidatat non proident, sunt in culpa qui officia dese runt mollit
+            anim id est laborum velit esse cillum dolore eu fugiat nulla pariatu
+            epteur sint occaecat
+          </p>
+        </div>
+        <ol class="AboutUs_Card_box">
```
Comment (andacenko) [inline index.html:397]: 'about-list'

File: index.html [hunk 14]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <li class="AboutUs_card">
+            <h3 class="AbautUs_card_header">High quality</h3>
```
Comment (andacenko) [inline index.html:399]: about-list__header

File: index.html [hunk 15]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          </li>
+          <li class="AboutUs_card">
+            <h3 class="AbautUs_card_header">Big community</h3>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and Ipsum
+              typesetting industry.
+            </p>
+          </li>
+        </ol>
+        <div class="AbautUs_button_box">
```
Comment (andacenko) [inline index.html:429]: delete

File: index.html [hunk 16]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <a href="#" class="Button_readMore">Read more</a>
```
Comment (andacenko) [inline index.html:430]: 'btn'

File: index.html [hunk 17]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        </div>
+      </section>
+      <!-- ----------------------------------------------------Our Team -->
+      <section class="Ourteam_section">
+        <div class="Ourteam_header_box">
```
Comment (andacenko) [inline index.html:435]: delete

File: index.html [hunk 18]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <h2 class="Ourteam_header">Our Team</h2>
```
Comment (andacenko) [inline index.html:436]: 'our-team__header'

File: index.html [hunk 19]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        </div>
+        <div class="Ourteam_card_box">
+          <div class="Ourteam_card">
+            <img src="" alt="" class="Ourteam_foto" />
+            <div class="Ourteam_info">
```
Comment (andacenko) [inline index.html:441]: delete (excessive)

File: index.html [hunk 20]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              <h3 class="Ourteam_name">Jason Smith</h3>
+              <p>Co - Founder</p>
+              <div class="Ourteam_socials_box">
```
Comment (andacenko) [inline index.html:444]: delete (excessive)

File: index.html [hunk 21]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+                    <a href="" class="socials"
+                      ><img src="" alt="Telegram" class="socials_logo"
+                    /></a>
+                    <a href="" class="socials"
+                      ><img src="" alt="LinkedIn" class="socials_logo"
+                    /></a>
+                    <a href="" class="socials"
+                      ><img src="" alt="Instagram" class="socials_logo"
+                    /></a>
+                  </div>
```
Comment (andacenko) [inline index.html:462]: Please change
```
<ul class="social-networks">
  <li>
    <a href="#">
       <img w='' h='' alt=''>
       <span class='accessibility'>name of social network</span> 
    </a>
  </li>
</ul>
```

File: index.html [hunk 22]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+      </section>
 
-      <section class="Blog_section"></section>
-      <section class="Abaut_us_section"></section>
-      <section class="Our_team_section"></section>
-      <section class="Faqs_section"></section>
-      <!-- Main -->
+      <!-- ----------------------------------------------------FAQs -->
+      <section class="Faqs_section">
+        <div class="Faqs_header_box">
```
Comment (andacenko) [inline index.html:584]: delete

File: index.html [hunk 23]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        </div>
+      </section>
 
-      <section class="Blog_section"></section>
-      <section class="Abaut_us_section"></section>
-      <section class="Our_team_section"></section>
-      <section class="Faqs_section"></section>
-      <!-- Main -->
+      <!-- ----------------------------------------------------FAQs -->
+      <section class="Faqs_section">
```
Comment (andacenko) [inline index.html:583]: 'faq'

File: index.html [hunk 24]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        <div class="Faqs_header_box">
+          <h2 class="Header_faqs">FAQs</h2>
```
Comment (andacenko) [inline index.html:585]: 'faq__header'

File: index.html [hunk 25]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        </div>
+        <div class="Faqs_question_aria">
```
Comment (andacenko) [inline index.html:587]: Please, use `<details> <summary>` strukture form open-close blocks:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

File: index.html [hunk 26]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <div class="Faqs_question_box">
+            <div class="Faqs_questionanswer_box">
+              <H3 class="Faqs_question">What are the NFTs?</H3>
```
Comment (andacenko) [inline index.html:590]: Wrong tag `<H3>` use `<h3>`

File: index.html [hunk 27]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        <div class="Footer_logo_box">
+          <Logo_box>
+            <img src="" alt="" class="Logoimg" />
+            <h2 class="Logo_header"></h2>
+          </Logo_box>
+          <p></p>
+        </div>
+        <div class="Footer_nav_box">
+          <h3>Subscribe</h3>
+          <nav class="Footer_nav">
```
Comment (andacenko) [inline index.html:644]: 'footer-nav'

File: index.html [hunk 28]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+            <ol class="Footer_nav_list">
```
Comment (andacenko) [inline index.html:645]: 'footer-nav__list'

File: index.html [hunk 29]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              <li class="Footer_nav_item"><a href="" class="item">home</a></li>
```
Comment (andacenko) [inline index.html:646]: 'footer-nav__item' and 'footer-nav__link'

File: index.html [hunk 30]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+        </div>
+      </section>
+      
     </main>
-    <footer class="footer"></footer>
+    <!-- ------------------------------------------------------------------------Footer -->
+    <footer class="footer">
+      <div class="Footer_bar_box">
+        <div class="Footer_logo_box">
+          <Logo_box>
```
Comment (andacenko) [inline index.html:636]: wrong tag

File: index.html [hunk 31]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+            <img src="" alt="" class="Logoimg" />
+            <h2 class="Logo_header"></h2>
```
Comment (andacenko) [inline index.html:638]: empty block

File: index.html [hunk 32]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+            <ol class="Footer_nav_list">
+              <li class="Footer_nav_item"><a href="" class="item">home</a></li>
+              <li class="Footer_nav_item"><a href="" class="item">home</a></li>
+              <li class="Footer_nav_item"><a href="" class="item">home</a></li>
+              <li class="Footer_nav_item"><a href="" class="item">home</a></li>
+            </ol>
+          </nav>
+        </div>
+        <div class="Footer_nav_box">
+          <h3>Subscribe</h3>
```
Comment (andacenko) [inline index.html:654]: 'footer__header'

File: index.html [hunk 33]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+              <li class="Footer_nav_item"><a href="" class="item">home</a></li>
+            </ol>
+          </nav>
+        </div>
+
+        <div class="Footer_info_box">
+          <div class="Footer_rigts_box">
+            <p>We are the best way to check the rarity of NFT collection.</p>
+          </div>
+          <div class="Footer_socials_box">
```
Comment (andacenko) [inline index.html:669]: social networks should be as above

File: index.html (hunk 32 above)
Comment (Federicofilini) [reply index.html:654]: i not understand ? what do you mean?

File: index.html [hunk 34]
```diff
@@ -4,17 +4,16 @@
...
   <body>
-    <!-- Header -->
+    <!-- --------------------------------------------------------------------------Header -->
     <header class="header">
-      <a id="logo__box">
-        <img src="./img/Group1.jpg" alt="" class="Logo__img" />
-        <h2 class="logo__text">BINABOX</h2>
-      </a>
+      <h1 class="logo__text">
+        <img src="./img/Logo111.svg" alt="Logo" class="logo__img" />BINABOX
```
Comment (andacenko) [inline index.html:14]: add width="" height="" for all `` and, please, answer for all my comment

File: index.css [hunk 35]
```diff
@@ -1,4 +1,5 @@
 @import url(./Reset.css);
+@import url(./Main.css);
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.css]: done

File: index.css [hunk 36]
```diff
@@ -1,4 +1,5 @@
...
 @font-face {
   font-family: Bakbak One;
   src: url(./fonts/Epilogue-Regular.ttf);
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.css]: done

File: index.html [hunk 37]
```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="./index.css" />
   </head>
   <body>
-    <!-- Header -->
+    <!-- --------------------------------------------------------------------------Header -->
     <header class="header">
       <a id="logo__box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 38]
```diff
@@ -9,7 +9,7 @@
...
         <img src="./img/Group1.jpg" alt="" class="Logo__img" />
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 39]
```diff
@@ -33,8 +33,11 @@ <h2 class="logo__text">BINABOX</h2>
       </nav>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html:32]: done

File: index.html [hunk 40]
```diff
@@ -33,8 +33,11 @@ <h2 class="logo__text">BINABOX</h2>
...
       <div class="Search_box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 41]
```diff
@@ -33,8 +33,11 @@ <h2 class="logo__text">BINABOX</h2>
...
         <div class="Search__field">
-          <img src="./img/search-md.svg" alt="Поиск" />
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html:36]: done

File: index.html [hunk 42]
```diff
@@ -33,8 +33,11 @@ <h2 class="logo__text">BINABOX</h2>
...
-          <input type="text" class="search-input" placeholder="Search" />
+          <img src="./img/search-md.svg" alt="Поиск" /><input
+            type="text"
+            class="search-input"
+            placeholder="Search"
+          />
         </div>
 
         <div class="rectangle"></div>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 43]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
         </div>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html:64]: done

File: index.html [hunk 44]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
       </div>
     </header>
+    <!-- -------------------------------------------------------------------------Main---- -->
     <main class="main">
+      <!-- ---------------------------------------------------Promo--- -->
+      <section class="Promo__section">
+        <div class="Promo__box_text">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 45]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+          <div class="Promo__header_box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 46]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+            <h2 class="Promo__header">
+              WITH<span class="Promo__header_logo">BINABOX</span>EXPLORE NFT
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 47]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+              COLLECTION
+            </h2>
+            <p>We are the best way to check the rarity of NFT collection</p>
+          </div>
+          <div class="Promo__number_zone">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 48]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+            <div class="Promo__number_box">
+              <span class="Promo__number">2240+</span>
+              <p>Total Iteam</p>
+            </div>
+            <div class="Promo__number_box">
+              <span class="Promo__number">1000+</span>
+              <p>Whitelist now</p>
+            </div>
+          </div>
+          <div class="Promo__grupe_buttons">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 49]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+            <div class="Promo__button_box">
+              <a href="#" class="button_ConnectWallet">Connect Wallet</a>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 50]
```diff
@@ -33,8 +33,11 @@ <h2 class="logo__text">BINABOX</h2>
       </nav>
       <div class="Search_box">
         <div class="Search__field">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 51]
```diff
@@ -33,8 +33,11 @@ <h2 class="logo__text">BINABOX</h2>
...
-          <img src="./img/search-md.svg" alt="Поиск" />
-          <input type="text" class="search-input" placeholder="Search" />
+          <img src="./img/search-md.svg" alt="Поиск" /><input
+            type="text"
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 52]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
         </div>
       </div>
     </header>
+    <!-- -------------------------------------------------------------------------Main---- -->
     <main class="main">
+      <!-- ---------------------------------------------------Promo--- -->
+      <section class="Promo__section">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 53]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+            <div class="Promo__number_box">
+              <span class="Promo__number">1000+</span>
+              <p>Whitelist now</p>
+            </div>
+          </div>
+          <div class="Promo__grupe_buttons">
+            <div class="Promo__button_box">
+              <a href="#" class="button_ConnectWallet">Connect Wallet</a>
+            </div>
+            <div class="Promo__button_box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 54]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+              <a href="#" class="button_WhiteList">Whitelist</a>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 55]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+            </div>
+            _
+          </div>
+        </div>
+        <img src="" alt="Image with a large blue cube" class="Promo__img" />
+        <div class="Promo_box_img">
+          <img src="" alt="Pattern at the bottom of the box" />
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 56]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+        </div>
+      </section>
+      <!-- ----------------------------------------------------Catalog -->
       <section id="Catalog_section">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 57]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+        <div class="Catalog_info_box">
+          <div class="Catalog_Lable_box">
+            <img src="" alt="Folder Picture" />
+            <h2>Catalog</h2>
+          </div>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html:110]: done

File: index.html [hunk 58]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+          <div class="Catalog_buttons_box">
+            <button>See all</button>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 59]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+            <!-- <span>Разделитель между кнопками</span -->
+            <button><img src="" alt="Галочка" /></button>
+            <button><img src="" alt="Галочка" /></button>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 60]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
+          </div>
+        </div>
         <article class="Banner__card">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 61]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
           <div class="Catalog__img__box">
             <img src="" alt="" class="Catalog__img" />
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 62]
```diff
@@ -49,8 +52,57 @@ <h3 class="Discord__text">Discord</h3></a
...
       <section id="Catalog_section">
+        <div class="Catalog_info_box">
+          <div class="Catalog_Lable_box">
+            <img src="" alt="Folder Picture" />
+            <h2>Catalog</h2>
+          </div>
+          <div class="Catalog_buttons_box">
+            <button>See all</button>
+            <!-- <span>Разделитель между кнопками</span -->
+            <button><img src="" alt="Галочка" /></button>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html [hunk 63]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
           </div>
         </article>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (Federicofilini) [reply index.html:195]: done

File: index.html (hunk 2 above)
Comment (Federicofilini) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (Federicofilini) [reply index.html:200]: Done

File: index.html (hunk 6 above)
Comment (Federicofilini) [reply index.html:211]: Done

File: index.html (hunk 7 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 8 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 12 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 64]
```diff
@@ -142,13 +194,507 @@ <h3 class="Card__name">Name Product</h3>
...
+          <p>
+            Duis aute irure dolor in reprehenderit in voluptate velit esse
+            cillum dolore eu fugiat nulla pariatur. Excepteur sint occae cat
+            cupidatat non proident, sunt in culpa qui officia dese runt mollit
+            anim id est laborum velit esse cillum dolore eu fugiat nulla pariatu
+            epteur sint occaecat
+          </p>
+        </div>
+        <ol class="AboutUs_Card_box">
+          <li class="AboutUs_card">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 14 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 16 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 11 above)
Comment (Federicofilini) [reply index.html]: done

File: index.html (hunk 15 above)
Comment (Federicofilini) [reply index.html]: done

File: index.html (hunk 17 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 18 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 20 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 21 above)
Comment (Federicofilini) [reply index.html:461]: Done

File: index.html (hunk 24 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 25 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 26 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 27 above)
Comment (Federicofilini) [reply index.html]: done

File: index.html (hunk 28 above)
Comment (Federicofilini) [reply index.html]: done

File: index.html (hunk 30 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 31 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 32 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 65]
```diff
@@ -4,17 +4,18 @@
...
   <body>
-    <!-- Header -->
+    <!-- --------------------------------------------------------------------------Header -->
     <header class="header">
-      <a id="logo__box">
-        <img src="./img/Group1.jpg" alt="" class="Logo__img" />
-        <h2 class="logo__text">BINABOX</h2>
-      </a>
+      <div class="logo">
+        <a href="#" id="lang" class="logo__link">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 66]
```diff
@@ -4,17 +4,18 @@
...
   </head>
   <body>
-    <!-- Header -->
+    <!-- --------------------------------------------------------------------------Header -->
     <header class="header">
-      <a id="logo__box">
-        <img src="./img/Group1.jpg" alt="" class="Logo__img" />
-        <h2 class="logo__text">BINABOX</h2>
-      </a>
+      <div class="logo">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 67]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+        <div class="promo_union_box">
+          <img
+            class="union-img"
+            src="./img/Patternsvg.svg"
+            alt="Pattern at the bottom of the box"
+          />
+        </div>
+      </section>
+      <!-- ----------------------------------------------------Catalog -->
+      <section id="catalog_section">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 68]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+            </p>
+          </li>
+        </ol>
+        <div class="abautus_button_box">
+          <a href="#" class="button_readmore">Read more</a>
+        </div>
+      </section>
+      <!-- ----------------------------------------------------Our Team -->
+      <section class="ourteam_section">
+        <div class="ourteam_header_box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 69]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+          <h2 class="ourteam_header">Our Team</h2>
+        </div>
+        <div class="ourteam_card_box">
+          <div class="ourteam_card">
+            <img src="" alt="" class="ourteam_foto" />
+            <div class="ourteam_info">
+              <h3 class="ourteam_name">Jason Smith</h3>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 70]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+      <!-- ----------------------------------------------------FAQs -->
+      <section class="faqs_section">
+        <div class="faqs_header_box">
+          <h2 class="header_faqs">FAQs</h2>
+        </div>
+        <div class="faqs_question_aria">
+          <details class="faq-item">
+            <summary class="faq-question">
+              <img src="" alt="Question icon" class="faq-question-icon" />
+              What are the NFTs?
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 71]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+            epteur sint occaecat
+          </p>
+        </div>
+        <ol class="aboutus_card_box">
+          <li class="aboutus_card">
+            <h3 class="abautus_card_header">High quality</h3>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and Ipsum
+              typesetting industry.
+            </p>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 72]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+      </section>
     </main>
-    <footer class="footer"></footer>
+    <!-- ------------------------------------------------------------------------Footer -->
+    <footer class="footer">
+      <div class="footer_bar_box">
+        <div class="footer_logo_box">
+          <div class="logo_box">
+            <img src="" alt="" class="logoimg" />
+            <h2 class="logo_header">BINABOX</h2>
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 73]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+          </div>
+          <p></p>
+        </div>
+        <div class="footer-nav__box">
+          <h3>Subscribe</h3>
+          <nav class="footer-nav">
+            <ol class="footer-nav__list">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 74]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+              <li class="footer-nav__item"><a href="" class="item">home</a></li>
+              <li class="footer-nav__item"><a href="" class="item">home</a></li>
+            </ol>
+          </nav>
+        </div>
+
+        <div class="footer_info_box">
+          <div class="footer_rigts_box">
+            <p>We are the best way to check the rarity of NFT collection.</p>
+            <div class="footer_socials_box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 34 above)
Comment (Federicofilini) [reply index.html:14]: Done

File: index.html (hunk 29 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html [hunk 75]
```diff
@@ -24,131 +25,689 @@ <h2 class="logo__text">BINABOX</h2>
...
+              <a href="#" class="button_whiteList">Whitelist now</a>
+            </div>
+          </div>
+        </div>
+        <img
+          src="./img/imagecube.svg"
+          alt="image with a large blue cube"
+          class="promo__img"
+        />
+        <div class="promo_union_box">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 23 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 22 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 19 above)
Comment (Federicofilini) [reply index.html]: Done

File: index.html (hunk 13 above)
Comment (Federicofilini) [reply index.html]: Done

**Review by andacenko — APPROVED** [review]

**Pull request merged**